scripts/collector/postgres/sql/schema_objects.sql

```diff
@@ -0,0 +1,121 @@
+with all_tables as (
+  select c.oid as object_id,
+    'TABLE' as object_category,
+    case
+      when c.relkind = 'r' then 'TABLE'
+      when c.relkind = 'S' then 'SEQUENCE'
```

Comment:
i get duplicate tables in my postgresql one for 'Table' one for 'Sequence'
should this be a 'Table'?